libraries/default.rb

```diff
@@ -0,0 +1,68 @@
+def collect_github_repositories(github_token, org)
+  if ::ObjectSpace.const_defined?('Chef')
+    # Chef exists! Lets make sure Octokit is installed.
+    chef_gem 'octokit'
+  end
```

Comment:
These are library functions, so the order in which they are called or whether they are called before the cookbook uploader recipe is not guaranteed.